library/core/src/iter/traits/iterator.rs

```diff
@@ -3462,10 +3461,9 @@ pub trait Iterator {
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
     #[rustc_diagnostic_item = "iter_cloned"]
-    fn cloned<'a, T: 'a>(self) -> Cloned<Self>
+    fn cloned<'a, T: Clone + 'a>(self) -> Cloned<Self>
     where
         Self: Sized + Iterator<Item = &'a T>,
-        T: Clone,
```

Comment:
I did mean for the comment I made about `where` clauses to generalize, because the `where` is not easy to elide here.